site/components/buttons/DeleteDisruptionButton.tsx

```diff
@@ -5,12 +5,14 @@ interface DeleteDisruptionButtonProps {
     disruptionId?: string;
     csrfToken?: string;
     buttonClasses?: string;
+    isTemplate?: string;
 }
 
 const DeleteDisruptionButton = ({
     disruptionId,
     csrfToken,
     buttonClasses,
+    isTemplate,
```

Comment:
We need to change the button text on Line 52 to be conditional and say "Delete template" if we are in the template flow.

Reply:
done